.changeset/soft-hotels-own.md

```diff
@@ -0,0 +1,12 @@
+---
+'@melt-ui/svelte': patch
+---
+
+[Tooling] Migrate coverage reporter from c8 to v8
+[Tests] Add tests for internal helpers:
```

Comment:
No need to update version if we're just adding tests, you can delete this file :)

Reply:
Done!